gprofiler/profilers/java.py

```diff
@@ -833,7 +843,7 @@ def start(self) -> None:
         super().start()
         try:
             # needs to run in init net NS - see netlink_kernel_create() call on init_net in cn_init().
-            run_in_ns(["net"], lambda: proc_events.register_exit_callback(self._proc_exit_callback), 1)
+            run_in_ns(["net"], lambda: proc_events.register_exit_callback(self._proc_exit_callback), 1)  # type: ignore
```

Comment:
Why is it `type ignore` here?

Generally - I suggest that, wherever it's not obvious, we add a comment explaining the type ignore. So the reader doesn't have to wonder, and also so we can slowly eliminate them as things progress and we solve typing issues.

Reply:
Removed some more ignores

Reply:
Yay